elasticsearch/values.yaml

```diff
@@ -56,6 +56,14 @@ resources:
     cpu: "1000m"
     memory: "2Gi"
 
+initResources: {}
```

Comment:
Can you also add this new variable into the readme please.

Reply:
Done.